torch_geometric/loader/mixin.py

```diff
@@ -29,20 +31,21 @@ def get_numa_nodes_cores() -> Dict[str, Any]:
     nodes = {}
     try:
         for node_path in numa_node_paths:
-            numa_node_id = int(osp.basename(node_path)[4:])
+            numa_node_id = int(os.path.basename(node_path)[4:])
```

Comment:
```suggestion
            numa_node_id = int(osp.basename(node_path)[4:])
```

https://github.com/pyg-team/pytorch_geometric/pull/8435